### ISSUE TITLE
Feature suggestion: Read current cell when F4 is pressed in Excel

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#7044]
When editing a cell in Excel, pressing F4 cycles through all combinations of absolute or relative references in the selected formula, or the current reference.  NVDA does not currently announce anything when this key is pressed to indicate the change in the formula.

It's a fairly advanced function, but still potentially handy (possibly blocked by the way Excel shows formulas being edited?)

**Comment by bhavyashah** [comment]
@feerrenrut Just to clarify, shouldn't this ticket be labelled as a feature instead of being marked an enhancement with a priority? I have tried to understand the distinction between a feature and enhancement multiple times, but I seem to be missing something again.

**Comment by feerrenrut** [comment]
@bhavyashah You're right, it should be. The approach we are now taking with regards to features / enhancements and priority levels is fairly recent. So there are likely quite a few issues out there that don't quite match the scheme. From practical standpoint, it doesn't matter too much. A priority on a feature or enhancement will basically be ignored by NV Access. We accept that its likely that there will be quite a few issues that should be labelled as `new feature` that have the `enhancement` label instead.
